docs/l2l_theme/css/base.css

```diff
@@ -1,6 +1,27 @@
 body {
     padding-top: 70px;
 }
+.col-md-9 h1::before {
```

Comment:
This makes functions manually added in the pydocmd.yml appear as classes.

For an example of the behaviour, see for example the generated docs for `l2l.clone_module`.

Reply:
hmm interesting, we'll probably need a better alternative in that case, since I am not sure how we can resolve that.